### PR TITLE
Add LDAP lookup config examples

### PR DESCRIPTION
**Pull request by blowdart**
Add LDAP lookup config examples for RC2

### CODE REVIEW

File: aspnetcore/security/authentication/windowsauth.md
Comment (JunTaoLuo) [inline aspnetcore/security/authentication/windowsauth.md:175]: Even though this API would "work" on Windows, it's mostly only relevant to Linux scenarios.

Comment (blowdart) [reply aspnetcore/security/authentication/windowsauth.md:175]: It's implied further up, @scottaddie should it get repeated down here? It's your doc really

**Review by JunTaoLuo — APPROVED** [review]

File: aspnetcore/security/authentication/windowsauth.md
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/windowsauth.md:155]: The style guide prefers `it's` over `it is`
I think it reads better with `it's` but it's a trivial nit :)

File: aspnetcore/security/authentication/windowsauth.md
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/windowsauth.md:175]:
```suggestion
Kerberos authentication doesn't provide any role information for an authenticated users. In order to add role and group information to a kerberos user, the kerberos authentication handler must be configured to retrieve the roles from an LDAP domain. The most basic configuration only specifies an LDAP domain to query against and will use the authenticated user's context to query the LDAP domain;
```

File: aspnetcore/security/authentication/windowsauth.md
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/windowsauth.md:188]:
```suggestion
Some configurations may require specific credentials to query the LDAP domain, this can also be specified in the options:
```

File: aspnetcore/security/authentication/windowsauth.md
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/windowsauth.md:203]:
```suggestion
By default the negotiate authentication handler resolves nested domains. In a large or complicated LDAP environment,  resolving nested domains may result in a slow lookup, or a lot of memory being used for each user. Nested domain resolution can be disabled using the `IgnoreNestedGroups` option.
```

**Review by Rick-Anderson — APPROVED** [review]

**Review by scottaddie — APPROVED** [review]
The PR looks great after incorporating Rick's suggested changes.

File: aspnetcore/security/authentication/windowsauth.md
Comment (Tratcher) [reply aspnetcore/security/authentication/windowsauth.md:175]:
```suggestion
Kerberos authentication on Linux or macOS doesn't provide any role information for an authenticated user. To add role and group information to a Kerberos user, the authentication handler must be configured to retrieve the roles from an LDAP domain. The most basic configuration only specifies an LDAP domain to query against and will use the authenticated user's context to query the LDAP domain:
```

**Comment by Rick-Anderson** [comment]
@blowdart  can you accept |reject my suggestions? Do we need to wait for @Tratcher  approval?

**Comment by blowdart** [comment]
It'd be @JunTaoLuo approval not Chris :)

File: aspnetcore/security/authentication/windowsauth.md
Comment (blowdart) [reply aspnetcore/security/authentication/windowsauth.md:155]: Bah. You americans and your contractions.

**Comment by blowdart** [comment]
OK I think that's it though. I can't approve my own PR ;)

**Pull request merged**
